stack/deprecating-interfaces.rst

```diff
@@ -51,20 +51,24 @@ Class and static methods should be decorated in the order given here::
 
     class Foo:
         @classmethod
-        @deprecated(reason="This has been replaced with `mm()`. Will be removed after v10.", category=FutureWarning)
+        @deprecated(reason="This has been replaced with `mm()`. Will be removed after v10.",
+	            version="v9.0", category=FutureWarning)
         def cm(cls, x):
             pass
 
         @staticmethod
-        @deprecated(reason="This has been replaced with `mm()`. Will be removed after v10.", category=FutureWarning)
+        @deprecated(reason="This has been replaced with `mm()`. Will be removed after v10.",
+	            version="v9.0", category=FutureWarning)
         def sm(x):
             pass
 
 The reason string should include the replacement API when available or explain why there is no replacement.
 The reason string will be automatically added to the docstring for the class or function; there is no need to change that.
 The reason string must also specify the version after which the method may be removed, as discussed in :ref:`code_removal`.
 
-You do not need to specify the optional version argument to the decorator since deprecation decorators are typically not added in advance of when the deprecation actually begins.
+The version argument to the decorator specifies the next release, when the deprecation will be in effect but the interface has not yet been removed.
+(Note that it is not required that developers inserting deprecation decorators know exactly what the next release will be; they may use the next major release in the version argument, even if it's a bit later than the actual first deprecation notice.)
```

Comment:
I'd remove the parantheses and "Note that", and rephrase the end as "even if it is later than the actual first deprecation notice"